ote/src/cljc/ote/transit_changes.cljc

```diff
@@ -220,15 +220,14 @@
   (excluding no-traffic days and static-holidays).
```

Comment:
`week=` could be renamed after it's not a comparison operation nowadays.
E.g. "week-traffic-changes?" or similar ?-function.

Reply:
But it still is a comparison operation. We are comparing the traffic of 2 weeks, while it is no longer so simple as comparing if they are equal, it's still comparing 2 weeks and atleast I've grown to like `week=`.